sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -284,17 +318,176 @@ a description of the layouts and detailed layout-specific rules. +
 'Stride' further qualifies how matrix elements are laid out in memory. It must be a
 scalar 'integer type' and its exact semantics depend on 'MemoryLayout'. +
  +
+'Memory Operand', if present, must begin with a _Memory Operand_ literal.
+If not present, it is the same as specifying the _Memory Operand_ None. +
+ +
+All the operands to this instruction must be dynamically uniform within every
+instance of the 'Scope' of the cooperative matrix. +
+ +
 1+|Capability: +
 *{capability_prefetch_name}*
-1+| 8+variable | {OpCooperativeMatrixPrefetchINTEL_token} | '<id>' +
+1+| 6+variable | {OpCooperativeMatrixPrefetchINTEL_token} | '<id>' +
 'Pointer' | '<id>' +
-'X offset' | '<id>' +
-'Y offset' | '<id>' +
 'Rows' | '<id>' +
 'Columns' | Literal +
 'Cache Level' | '<id>' +
 'MemoryLayout' | Optional '<id>' +
-'Stride' |
+'Stride' | Optional +
+'Memory Operand' |
+|=====
+
+[cols="1,1,10*3",width="100%"]
+|=====
+11+|[[OpCooperativeMatrixLoadCheckedINTEL]]*OpCooperativeMatrixLoadCheckedINTEL* +
+ +
+Load a cooperative matrix through a pointer. Global matrix size might be not multiple the size of
+the two-dimentional region that is being loaded, in this case the out-of-bounds elements are
+set to 0. +
```

Comment:
It would be useful to define what is meant by the "global matrix" somewhere.

It's not clear to me why being a multiple of the size of the 2D region being loaded is relevant (or not, in the case of this instruction).  Isn't this really about going "off the edge", which could happen even if the block size evenly divides the matrix size, if the X offset or Y offset set appropriately.

Also, there's a minor typo here:

```suggestion
Load a cooperative matrix through a pointer. Global matrix size might be not multiple the size of
the two-dimensional region that is being loaded, in this case the out-of-bounds elements are
set to 0. +
```